pages/events/[type].js

```diff
@@ -31,6 +35,9 @@ export default function EventsPage() {
   if (!session) {
     return <Login />;
   }
+  if (type != "wishlist" && type != "upcoming" && type != "latest") {
```

Comment:
is there a reason you don't work with strict inequality here? 🤔 

```suggestion
  if (type !== "wishlist" && type !== "upcoming" && type !== "latest") {
```

Reply:
Yes, the reason is that I forgot to use it 😄 Done